package.json

```diff
@@ -3,7 +3,6 @@
   "version": "2.0.2",
   "description": "A plugin to make Backbone.js keep track of nested attributes.",
   "scripts": {
-    "install": "bower install",
     "test": "grunt"
```

Comment:
I would rather change this line to be `bower install && grunt` rather than making the install grunt task above.
Unless you update the task to only do `bower install` the first time.